hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/HadoopDirTreeGenerator.java

```diff
@@ -154,14 +155,14 @@ private void createDir(long counter) throws Exception {
       createSubDirRecursively(dir, 1, 1);
     }
     String message = "Successfully created directories & files. Total Dirs " +
-            "Count=" + totalDirsCnt.get() + ", Total Files Count=" +
-            timer.getCount();
+        "Count=" + totalDirsCnt.get() + ", Total Files Count=" +
+        timer.getCount();
     print(message);
   }
 
   private void createSubDirRecursively(String parent, int depthIndex,
                                        int spanIndex)
-          throws Exception {
+      throws Exception {
```

Comment:
Can be merged with previous line. (removing the vertical-align)

Reply:
Sorry about that, Ill make the necessary changes !!